tests/test_forcefields.py

```diff
@@ -63,3 +63,21 @@ def test_loading_forcefield_from_file():
             with pytest.raises(ValueError, match="Unsupported charge handler"):
                 Forcefield.load_from_file(bad_ff)
         assert len(w) == 0
+
+
+def test_load_default():
+    """assert that load_default() is an alias for load_from_file(DEFAULT_FF)"""
+
+    ref = Forcefield.load_from_file(constants.DEFAULT_FF)
+    test = Forcefield.load_default()
+
+    # ref == test evaluates to false, so assert equality of smirks lists and parameter arrays manually
+
+    ref_handles = ref.get_ordered_handles()
+    test_handles = test.get_ordered_handles()
+
+    assert len(ref_handles) == len(test_handles)
+
+    for (ref_handle, test_handle) in zip(ref.get_ordered_handles(), test.get_ordered_handles()):
+        assert ref_handle.smirks == test_handle.smirks
+        assert (ref_handle.params == test_handle.params).all()
```

Comment:
nit: should we prefer `np.testing.assert_array_equal`? (mainly for more helpful message on failure)

Reply:
Yes! Fixed in https://github.com/proteneer/timemachine/pull/989/commits/d5476858efc993ca96e7ac581c1e5583fc0a03a8